test/k6/src/tests/orders_org_no.js

```diff
@@ -0,0 +1,263 @@
+  success = check(response, {
+    "GET email notifications. Status is 200 OK": (r) => r.status === 200,
+  });
+
+  addErrorCount(success);
+  if (!success) {
+    // only continue to parse and check content if success response code
+    stopIterationOnFail(success);
+  }
+
+  success = check(JSON.parse(response.body), {
+    "GET email notifications. OrderId property is a match": (
+      notificationSummary
+    ) => notificationSummary.orderId === orderId,
+  });
+}
+
+// 06 - GET email notifications summary again after one minute for verification
+function TC06_GetEmailNotificationSummaryAgainAfterOneMinuteForVerification(data, orderId) {
+  sleep(60); // Waiting 1 minute for the notifications to be generated
+  var response, success;
+  response = notificationsApi.getEmailNotifications(orderId, data.token);
+  success = check(response, {
+    "GET email notifications summary again after one minute for verification. Status is 200 OK": (r) => r.status === 200,
+  });
+
+  addErrorCount(success);
+  if (!success) {
+    // only continue to parse and check content if success response code
+    stopIterationOnFail(success);
+  }
+
+  success = check(JSON.parse(response.body), {
+    "GET email notifications summary again after one minute for verification. OrderId property is a match": (
+      notificationSummary
+    ) => notificationSummary.orderId === orderId,
+    "GET email notifications summary again after one minute for verification. At least one notification has been generated": (
+      notificationSummary
+    ) => notificationSummary.generated > 0,
+    "GET email notifications summary again after one minute for verification. At least one notification is in the notifications array": (
+      notificationSummary
+    ) => notificationSummary.notifications.length > 0,
+    "GET email notifications summary again after one minute for verification. Recipient organization number is a match": (
+      notificationSummary
+    ) => notificationSummary.notifications[0].recipient.organizationNumber === data.emailOrderRequest.recipients[0].organizationNumber,
+    "GET email notifications summary again after one minute for verification. Recipient email address found in the contact lookup for the given organization number": (
+      notificationSummary
+    ) => notificationSummary.notifications[0].recipient.emailAddress.length > 0,
+  });
+}
+
+/*
+ * 01 - POST email notification order request
+ * 02 - GET notification order by id
+ * 03 - GET notification order by senders reference
+ * 04 - GET notification order with status
+ * 05 - GET email notification summary
+ * 06 - GET email notifications summary again after one minute for verification
+ */
+export default function (data) {
+  try {
+    if (data.runFullTestSet) {
+      var selfLink = TC01_PostEmailNotificationOrderRequest(data);
+      let id = selfLink.split("/").pop();
+      TC02_GetNotificationOrderById(data, selfLink, id);
```

Comment:
not sure I see the point of duplicating test caces 2-5 from the existing orders_email tests. Recipient being an organization doesn't change any of the logic related to retrieving an order.

Reply:
I also think that we can remove some of the duplicated tests.

Reply:
I do agree! Will remove them today! 👍

Reply:
I do agree that 02-04 we don't need! However we do need the 05 for the use case test, don't we? 🤔